SWLOR.Game.Server/Feature/PerkDefinition/RangedPerkDefinition.cs

```diff
@@ -118,7 +156,7 @@ private void PointBlankShot()
                 .Name("Point Blank Shot")
 
                 .AddPerkLevel()
-                .Description("Grants +1 to your attack roll and damage when your target is within 15 feet.")
+                .Description("While a target is within 5 meters / 15 feet, you gain +5 accuracy and +1 DMG.")
```

Comment:
Everything in the game references meters, so let's remove the section about feet from here for consistency.